wandb/integration/xgboost/xgboost.py

```diff
@@ -0,0 +1,181 @@
+# -*- coding: utf-8 -*-
+"""
+xgboost init
+"""
+
+import json
+import wandb
+import warnings
+import xgboost as xgb
+from typing import cast
+
+
+MINIMIZE_METRICS = [
+    "rmse",
+    "rmsle",
+    "mae",
+    "mape",
+    "mphe",
+    "logloss",
+    "error",
+    "error@t",
+    "merror",
+    "mlogloss",
+]
+
+MAXIMIZE_METRICS = ["auc", "aucpr", "ndcg", "map", "ndcg@n", "map@n"]
+
+
+def wandb_callback():
+    """
+    Old style callback that will be deprecated in favor of WandbCallback. Please try the new logger for more features. 
+    """
+    warnings.warn(
+        "wandb_callback will be deprecated in favor of WandbCallback. Please use WandbCallback for more features.",
+        UserWarning,
+        stacklevel=2,
+    )
+
+    def callback(env):
+        for k, v in env.evaluation_result_list:
+            wandb.log({k: v}, commit=False)
+        wandb.log({})
```

Comment:
can you add an old style test in yea also.

Reply:
Sure. it will be a separate test right?

Reply:
yes.  just a simple one, probably one in our examples docs somewhere.

Reply:
I have added a simple one as `yea` test itself. If that's not okay let me know.